addons/binding/org.openhab.binding.plugwise/src/main/java/org/openhab/binding/plugwise/handler/PlugwiseRelayDeviceHandler.java

```diff
@@ -0,0 +1,576 @@
+                updateState(CHANNEL_STATE, OnOffType.OFF);
+                break;
+            case POWER_LOG_INTERVAL_SET_ACK:
+                logger.debug("Received ACK for power log interval set of {} ({}) ", deviceType, macAddress);
+                updateMeasurementInterval = false;
+                break;
+            default:
+                logger.debug("{} ({}) {} acknowledgement", deviceType, macAddress, extensionCode);
+                break;
+        }
+
+        boolean newConfigurationPending = isConfigurationPending();
+
+        if (oldConfigurationPending != newConfigurationPending && !newConfigurationPending) {
+            Configuration newConfiguration = editConfiguration();
+            newConfiguration.put(CONFIG_PROPERTY_UPDATE_CONFIGURATION, false);
+            updateConfiguration(newConfiguration);
+        }
+
+        updateStatusOnDetailChange();
+    }
+
+    private void handleCalibrationResponse(PowerCalibrationResponseMessage message) {
+        boolean wasCalibrated = isCalibrated();
+        calibration = message.getCalibration();
+        logger.debug("{} ({}) calibrated: {}", deviceType, macAddress, calibration);
+        if (!wasCalibrated) {
+            if (isRecentLogAddressKnown()) {
+                updateEnergy();
+            } else {
+                updateInformation();
+            }
+            sendFastUpdateMessage(new PowerInformationRequestMessage(macAddress));
+        }
+    }
+
+    private void handleClockGetResponse(ClockGetResponseMessage message) {
+        String clock = message.getTime();
+        updateState(CHANNEL_CLOCK, new StringType(clock));
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.debug("Handling command '{}' for {} ({}) channel '{}'", command, deviceType, macAddress,
+                channelUID.getId());
+        try {
+            switch (channelUID.getId()) {
+                case CHANNEL_STATE:
+                    if (command instanceof OnOffType) {
+                        if (configuration.getPowerStateChanging() == PowerStateChanging.COMMAND_SWITCHING) {
+                            OnOffType onOff = (OnOffType) command;
+                            pendingPowerStateChange = new PendingPowerStateChange(onOff);
+                            handleOnOffCommand(onOff);
+                        } else {
+                            OnOffType onOff = configuration.getPowerStateChanging() == PowerStateChanging.ALWAYS_ON
+                                    ? OnOffType.ON : OnOffType.OFF;
+                            logger.debug("Ignoring {} ({}) power state change (always {})", deviceType, macAddress,
+                                    onOff);
+                            updateState(CHANNEL_STATE, onOff);
+                        }
+                    }
+                    break;
+                default:
+                    break;
+            }
+        } catch (Exception e) {
```

Comment:
Removed the catch. It is not mission critical and the framework can also show a nice error message when the command fails.